client/core/core.go

```diff
@@ -6172,15 +6188,38 @@ func (c *Core) loadDBTrades(dc *dexConnection, crypter encrypt.Crypter, failed m
 		}
 		if baseFailed {
 			errs.add("could not complete order %s because the wallet for %s cannot be used", trade.token(), unbip(base))
-			continue
 		}
 		if quoteFailed {
 			errs.add("could not complete order %s because the wallet for %s cannot be used", trade.token(), unbip(quote))
-			continue
```

Comment:
Happy this is being addressed.  Just not loading the orders for wallets that aren't connected or cannot unlock was poor.

I'm realizing that with my latest merged work where `walletSet` is now just the `xcWallets`, we seem to be able to set `trackedTrade.wallets` in `loadDBTrades` when initially constructing with `newTrackedTrade` instead of later in `resumeTrades` (and not failing on connect or unlock errors, rather just using `c.wallet` instead of `c.connectedWallet` and `unlock`, which can still be attempted before `resumeTrades`).  That would mean that "ready" trades include any trade with a complete `walletSet` **regardless of if there are wallet connection or unlocking issues.**  And really, don't we want all trades listed in the trades map so the user is aware and can fix the issue?

Reply:
Oh, I just read your comment in https://github.com/decred/dcrdex/pull/1903#discussion_r1019525129 saying you intend to list the failed ones.  Still see if my comment makes sense w.r.t. setting the `walletSet` field of the `trackedTrades` in `loadDBTrades` if possible.  Would that improve anything, or just make spam the logs with errors on each `tick`?

Reply:
Yeah, there will be a constant spam of error logs for each trade that failed to load on each `tick`. Does the approach I wrote in that comment sound good?

Reply:
I'm not sure yet, but I think we want `trackedTrade`s with wallet issues to still be loaded so you can cancel the orders.  There are three general conditions in `checkIfTradesAreReady` that make a trade "not ready" or "failed to load":

1. No wallet definition for one of the assets, hence cannot make an `xcWallet`.  This is when `c.wallet` errors in `c.connectedWallet`.
2. Wallet not connected, but there was a wallet definition.  This is when `c.connectAndUpdateWallet` errors in `c.connectedWallet`.
3. `(*xcWallet).Unlock` errors

The first is resolvable by creating the wallet, but I don't know how this would happen.

The second is resolvable by starting the external wallet, or perhaps "recovering" a busted native wallet.

The third is resolvable by reconfiguring the wallet with the new password.

Each of the resolutions would need to trigger a re-check of the `failedTrades`.  I don't know that `(*Core).OpenWallet` would cover any of the above scenarios.

In all but the first of those three causes we can make a complete `walletSet` for the `trackedTrade.wallet` fields.  And only trades with actively negotiating swaps would actually spam with errors on `tick` I think.  If the order is still booked, regardless of whether it has active swaps, being able to cancel that order without resolving the wallet situation would be desirable.

Reply:
> In all but the first of those three causes we can make a complete `walletSet` for the `trackedTrade.wallet` fields.

That first one would be a doozy.

As long as we can make a `walletSet`, I think the order can go in the `trades` map. Where in tick would we be logging? After `refreshUnlock`? Could also set a flag on the `trackedTrade` to bail early in `tick`, right after `t.mtx.RLock()`, I guess.

Reply:
These types of logs would show up in every tick: `not checking if order %s, match %s is redeemable because %s wallet is locked or disabled`.

I've added a flag to `trackedTrade` that blocks the ticks. If the `walletSet` cannot be created, we get the same behavior as before, but all trades are added to the trades map now. Let me know what you guys think.